src/main/java/com/liberty52/product/global/adapter/dalle/dto/PapagoDto.java

```diff
@@ -0,0 +1,102 @@
+package com.liberty52.product.global.adapter.dalle.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.liberty52.product.global.adapter.dalle.DallEApiClient;
+import com.liberty52.product.global.exception.internal.DallEApiClientExceptions;
+import lombok.Getter;
+
+import java.util.List;
+
+public class PapagoDto {
```

Comment:
```suggestion
public class DallEDto {
```

DallE 모듈에서 PapagoDto라고 해서 조금 헷갈려요;

Reply:
실수네여